lib/cryptoexchange/exchanges/vinex/services/market.rb

```diff
@@ -0,0 +1,51 @@
+module Cryptoexchange::Exchanges
+  module Vinex
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super(ticker_url)
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Vinex::Market::API_URL}/markets"
+        end
+
+        def adapt_all(output)
+          output["data"].map do |ticker|
+            base, target = ticker['symbol'].split("_")
+            market_pair  = Cryptoexchange::Models::MarketPair.new(
+              base:   base,
+              target: target,
+              market: Vinex::Market::NAME
+            )
+            adapt(ticker, market_pair)
+          end
+        end
+
+        def adapt(output, market_pair)
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Vinex::Market::NAME
+
+          ticker.last      = NumericHelper.to_d(output['lastPrice'])
+          ticker.high      = NumericHelper.to_d(output['high24h'])
+          ticker.low       = NumericHelper.to_d(output['low24h'])
+          ticker.volume    = NumericHelper.to_d(output['volume'])
+          ticker.change    = NumericHelper.to_d(output['change24h'])
+
+          ticker.timestamp = output['updatedAt']
```

Comment:
lets assign this to `nil` and not trust the timestamp. its been giving us issues for other exchanges